pycroft/lib/swdd.py

```diff
@@ -13,7 +14,7 @@
 swdd_hmac_key = os.environ.get('SWDD_HASH_KEY')
 
 
-def get_swdd_person_id(first_name: str, last_name: str, birthdate: str) -> int | None:
```

Comment:
Please do API changes (such as the datatype of a passed argument into this lib function) in a separate commit in the future to make verification / review easier. Fine for now, I'll just have to check all the usages.

Reply:
Checked everything, looks good.